src/utils/formatter.py

```
@@ -4,201 +4,216 @@
 This module provides utilities for formatting responses for Discord.
 """
 
+from typing import Any, Dict, List
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import

(F401)